src/pages/theEyePages/changeSetting/ChangeSetting.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Typography, Grid } from '@material-ui/core';
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import GenericNavbar from '../../../components/generic/GenericNavbar/GenericNavbar';
+import Button from '../../../components/generic/buttons/GenericButton'
+import { TextInput, SelectInput, Checkboxes } from '../../../components/formInputs/FormInputs';
+import Container from '../../../components/generic/container/Container';
+import Tooltips from '../../../components/tooltip/Tooltips';
+import '../theEye.css';
+
+const ChangeSetting = () => {
+    const [name, setName] = React.useState('');
```

Comment:
Might be worth moving `useState` as it's own import (`import React, { useState } from 'react';`) and then just using `useState` over `React.useState` here?

Reply:
You could do that but wouldn't it be easier to just keep the useState inside of it's intended component for simplicities sake?